src/doc/python_scripting/functions.rst

```diff
@@ -12175,15 +12178,42 @@ WriteConfigFile
 WriteScript
 -----------
 
+**Synopsis:**
+
+::
+
+  WriteScript(f)
+
+f : file
+    The python file object that will be written to.
+
```

Comment:
Where can I find the return type for this function? I feel like I looked everywhere.

Reply:
I think its in `src/visitpy/visit_utils/src/builtin/writescript.py`.

Ultimately, the VisIt python interface is a mixture of C/C++ Extension modules to Python, pure python functions, and other python modules (e.g. `numpy`, `pyside`, etc.). `WriteScript` is a pure python function.